lifetimes/generate_data.py

```diff
@@ -61,6 +61,66 @@ def beta_geometric_nbd_model(T, r, alpha, a, b, size=1):
     return df.set_index('customer_id')
 
 
+def beta_geometric_nbd_model_transactional_data(T, r, alpha, a, b, start_date='2019-1-1', size=1):
```

Comment:
should we be opinionated about the start date?

Reply:
In fact, is it possible to generalize this to any time units (not just days)?

Reply:
Yes, any valid date format can be used here. For example `'2019-1-1 12:30:21'`.